trillian/migrillian/core/config.go

```diff
@@ -53,3 +55,33 @@ func ValidateConfig(cfg *configpb.MigrationConfig) error {
 	}
 	return nil
 }
+
+// ValidateConfig verifies that MigrillianConfig is correct. In particular:
+// - The log backends have distinct non-empty names and backend specs.
+// - Migration configs are valid (as per ValidateMigrationConfig).
+// - Migration configs specify backend names present in the set of backends.
+// - Each migration config has a unique (backend, tree ID) pair.
+// Returns a map from log backend names to the corresponding LogBackend's.
+func ValidateConfig(cfg *configpb.MigrillianConfig) (ctfe.LogBackendMap, error) {
+	lbm, err := ctfe.BuildLogBackendMap(cfg.Backends)
+	if err != nil {
+		return nil, err
+	}
+	// Check that logs all reference a defined backend and there are no duplicate
```

Comment:
Can you clarify the comment that the duplicate log id check is per backend.

Reply:
Done.